docs/source/en/guides/collections.md

```diff
@@ -58,6 +58,45 @@ A `note` can also be attached to the item. This is useful to add additional info
 
 In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
 
+## List collections
+
+We can also get multiple collections with [`list_collection`]. It allows us to retrieve collections and filter them with certain parameters. Let's list all collections of the user `teknium`.
+```py
+>>> from huggingface_hub import list_collections
+
+>>> collections = list_collections(owner="teknium")
+```
+This returns an iterable of `Collection` objects. We can iterate over it to print number of upvotes for each collections.
+```py
+>>> for i in collections:
+...   print("Number of upvotes:", i.upvotes)
+Number of upvotes: 1
+Number of upvotes: 5
+```
+
+It is possible to do more advanced filtering. Let's get all model collections, sort by trending, and limit the count to 5.
+```py
+>>> collections = list_collections(item="models", sort="trending", limit=5):
+>>> for i in collections:
+...   print("Number of upvotes:", i.upvotes)
+Number of upvotes: 39
+Number of upvotes: 178
+Number of upvotes: 21
+Number of upvotes: 11
+Number of upvotes: 14
```

Comment:
```suggestion
>>> collections = list_collections(item="models/TheBloke/OpenHermes-2.5-Mistral-7B-GGUF", sort="trending", limit=5):
>>> for collection in collections:
...   print(collection.slug)
teknium/quantized-models-6544690bb978e0b0f7328748
AmeerH/function-calling-65560a2565d7a6ef568527af
PostArchitekt/7bz-65479bb8c194936469697d8c
gnomealone/need-to-test-652007226c6ce4cdacf9c233
Crataco/favorite-7b-models-651944072b4fffcb41f8b568
```

(adapting to filter by specific item + showcase that we can also get the slug (to show something different than the previous example))